vendor/k8s.io/kubernetes/pkg/kubectl/resource_printer.go

```diff
@@ -844,39 +868,8 @@ func printReplicaSetList(list *extensions.ReplicaSetList, w io.Writer, options P
 	return nil
 }
 
-func printCluster(c *federation.Cluster, w io.Writer, options PrintOptions) error {
```

Comment:
@rhcarvalho Hm, I had assumed these had been removed as part of the rebase update, but they still [seem to be there in master](https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/kubectl/resource_printer.go#L35). Thanks for catching this, I will take a look right now